include/onnxruntime/core/providers/tensorrt/tensorrt_provider_options.h

```diff
@@ -88,4 +88,6 @@ struct OrtTensorRTProviderOptionsV2 {
 
   const char* trt_engine_cache_prefix{nullptr};  // specify engine cache prefix
   int trt_engine_hw_compatible{0};               // Enable hardware compatibility. Default 0 = false, nonzero = true
+  const char* trt_nodes_to_exclude{nullptr};     // Exclude specific nodes from running on TRT e.g. "NonMaxSuppression,NonZero,RoiAlign".
```

Comment:
ops or nodes?

Reply:
nodes implies you can exclude specific nodes , rather than all instances of an op

Reply:
It's ops. Will update the code to use ops.